administrator/components/com_admin/sql/updates/mysql/5.3.0-2025-02-22.sql

```diff
@@ -0,0 +1,3 @@
+-- Add plg_extension_joomlaupdate plugin
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `ordering`, `state`) VALUES
+(0, 'plg_extension_joomlaupdate', 'plugin', 'joomlaupdate', 'extension', 0, 1, 1, 0, 1, '', '', '', 4, 0);
```

Comment:
```suggestion
(0, 'plg_extension_joomlaupdate', 'plugin', 'joomlaupdate', 'extension', 0, 1, 1, 0, 1, '', '', '', 0, 0);
```
In update SQL scripts we should use ordering zero so new plugins are shown at the top.